### ISSUE TITLE
CentOS 8 EOL plan

### ISSUE THREAD

**Issue by Habbie** [PowerDNS/pdns#11056]
On 31 December 2021, CentOS 8 will go [EOL](https://www.centos.org/centos-linux-eol/). At that point, our building for RHEL8 and derivatives (Oracle, Rocky, Alma) will have to switch from CentOS 8 to one of those.

It turns out that we hardcoded `centos` in many places - in our build system, in our repo paths, in the names of .repo files. It's also all over the text in https://repo.powerdns.com/.

Proposed migration plan:
* all paths containing `centos` get replaced (with symlinks for existing users) by a new descriptor, perhaps `el`
* we add a generic `el8` (and `el7`) target to `pdns/builder-support` so that we can, in the future, switch build environments easily
* we update Buildbot config to use this generic target, and drop packages in the generic dirs (which the centos-dirs will be symlinked to, as mentioned above)
* we update https://repo.powerdns.com/ to link to the new `el` things, with some more words on what `el` means and what we do and do not promise about where and how we build

Naming proposals, repeated from above:
* replace `centos` by `el` in repo paths
* `build.sh` targets: `el-7` and `el-8`

**Comment by neilcook** [comment]
We're switching to oracle right?

**Comment by Habbie** [comment]
Yes, we'll switch to oracle, but label as many things as possible with something generic like "el8" so it is clear that the packages work on all those distributions.

**Comment by omoerbeek** [comment]
@Habbie I think this can be closed?

**Comment by Habbie** [comment]
I also think so!
